system76driver/gtk.py

```diff
@@ -21,7 +21,7 @@
 Gtk UI.
 """
 
-import platform
+import distro
```

Comment:
`distro` is not installed by default on Ubuntu/Pop 18.04. We need to add the `python3-distro` package to the `Depends:` and `Build-Depends:` sections in `/debian/control` for this to work on these older platforms.

Reply:
I'm not familiar with the debian build system since I use Arch, but I did as you asked according to the debian control documentation here: https://www.debian.org/doc/debian-policy/ch-controlfields.html#syntax-of-control-files. I can't test this though so let me know if this works for you.